app/models/instrument.rb

```diff
@@ -14,6 +14,13 @@
 
 class Instrument < ActiveRecord::Base
 
+  validates :name, presence: true
+  validates :years_played, presence: true
+  validates :experience_level, presence: true
+  validates :is_primary, presence: true
```

Comment:
while you're at it, can you change is_paid's validation on the lesson model to that one too